src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests_Regex.vb

```diff
@@ -33,23 +33,26 @@ end class
             End Using
         End Function
 
-        <MemberData(NameOf(AllCompletionImplementations), Skip:="https://github.com/dotnet/roslyn/issues/33852")>
+        <MemberData(NameOf(AllCompletionImplementations))>
         <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
         Public Async Function TestCaretPlacement(completionImplementation As CompletionImplementation) As Task
             Using state = TestStateFactory.CreateVisualBasicTestState(completionImplementation,
                 <Document><![CDATA[
 Imports System.Text.RegularExpressions
 class c
     sub goo()
-        dim r = New Regex("$$")
```

Comment:
@CyrusNajmabadi Before this change, why was the `dim` turning into `Dim`? The auto-formatting behavior is not being tested at all in the new version, is that a problem?

Reply:
@ivanbasov Did you have to make it `Dim` to get the test to pass?

Reply:
If we're trying to prove that line commit / case correction _doesn't_ run, perhaps we want it lowercase before & after?

Reply:
I believe the test harness in `master-vs-deps` excludes FormatCommandHandler which makes the `dim `to `Dim `replacement in `master`. We can make both `dim` or both `Dim`. I do not see a reason to change to `dim `now. 
@CyrusNajmabadi  please confirm
@dpoeschl please sign off

Reply:
> I believe the test harness in master-vs-deps excludes FormatCommandHandler

I'm still confused. Is that intentional? Were we previously testing nuances of interactions (or lack of interactions) between the two features (such as override completion triggering formatting, etc.) and now we've lost that?

Reply:
Yeah, this sin't great.  The intent of the CommandHandler tests was to actually try to be as close to an 'integration test' as we could get, without actually suffering the pain of really bein an integration test.  It allows us ot really ensure the end-to-end experience is workign properly.  This was necessary as so many tests actually unit-tested lower levels, but ended up being mostly useless since things still didn't work properly once all hooked up.

Note: the usage of lowercase in VB is very intentional and is a common way we use to ensure that things like 'case correction' don't run in these scenarios unexpectedly.

Reply:
@CyrusNajmabadi , @dpoeschl 
OK. I made a deeper review.
1. The change `dim` to `Dim` is performed by `CodeCleanup` in VB.
2. It can be triggered only if the line  of the current point changes: https://github.com/dotnet/roslyn/blob/5f12f54873000496a45f76beea5d7503865c6ae6/src/EditorFeatures/VisualBasic/LineCommit/CommitBufferManager.vb#L221
3. This one is called from https://github.com/dotnet/roslyn/blob/fab7134296816fc80019c60b0f5bef7400cf23ea/src/EditorFeatures/VisualBasic/LineCommit/CommitViewManager.vb#L42
4. Editor fixed an issue here  Editor bits are updated in `master-vs-deps` but not in `master`.
5. Previously, the old point position was always 0. Therefore, line changed in most of cases where it should not change. Therefore, we previously made `CodeCleanup` when we should not do it.
6. The current test scenario does not involve line changes. Therefore, it should be either `dim-dim` or `Dim-Dim` but not `dim-Dim`.

Please let me know if you see more issues. Otherwise, let us signoff/merge it.

Reply:
Ok.  Can you keep it as 'dim' then?